### DISCUSSION TITLE
[build] Pyodide Build

### DISCUSSION THREAD

**Discussion by kishorekaruppusamy** [pyodide/pyodide#4275] [Q&A]
I am trying to build a pyodide package for onnxruntime
by using ```pyodide skeleton pypi <package-name>``` this command we can generate yaml file but my problem is it automatically searches .tar.gz in pypi repo but in their there is ``` no tar ball is provided ```

can i able to use my own tar ball here instead of fetching tar.gz file from pypi so that i can easily generate pyodide build or can i able to build package using package without hosted in pypi repo ??

URGENCY : ```Urgent```

**Comment by ryanking13** [comment]
Yes, it is possible. See: https://pyodide.org/en/stable/development/building-and-testing-packages.html
